reconstruct/reconstruct.go

```diff
@@ -236,18 +177,18 @@ func Fix_fc(root *tree_newick.NTree, node *tree_newick.NTree) []float64 {
 }
 
 //loop over the nodes of the tree to fix each node and append the most probable base to the Fasta
-func Loop_nodes(root *tree_newick.NTree) {
-	leafs := Get_leaf(root)
-	branches := Get_branch(root)
-	for j := 0; j < len(leafs[0].Fasta.Seq); j++ {
-		for i := 0; i < len(leafs); i++ {
-			leafs[i].State = int(leafs[i].Fasta.Seq[j])
+func LoopNodes(root *expandedTree.ETree) { //what is fixing the nodes doing?
+	leaves := expandedTree.GetLeaves(root)
+	branches := expandedTree.GetBranch(root)
+	for j := 0; j < len(leaves[0].Fasta.Seq); j++ {
+		for i := 0; i < len(leaves); i++ {
+			leaves[i].State = int(leaves[i].Fasta.Seq[j])
 		}
 	}
 	for j := 0; j < len(branches); j++ {
```

Comment:
Change term from j to something else. Should be inside the J loop above so each column is handled independently in leaves and branches